docs/.link-check-errors.txt

```diff
@@ -1,4 +1,6 @@
 Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
+- https://mui.com/joy-ui/react-list/#actionable
+- https://mui.com/joy-ui/react-list/#selected
```

Comment:
This should be fixed because these links do not exist anymore. Can you search for these links in the project and replace with the new one?